specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Subscriptions.json

```diff
@@ -412,6 +486,23 @@
                 "modelAsString": true
             }
         },
+        "MoveSubscriptionsDefinition": {
+            "description": "The move subscriptions action definition",
+            "type": "object",
+            "properties": {
+                "targetDelegatedProviderOffer": {
+                    "description": "The delegated provider offer identifier (from the Admin context) that the subscriptions to be moved to.",
+                    "type": "string"
+                },
+                "resources": {
+                    "description": "A collection of subscriptions to move to the target delegated provider offer.",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                }
```

Comment:
are these two properties `targetDelegatedProviderOffer` and  `resources` required? if yes please mark so.